config.in

```diff
@@ -16,7 +16,8 @@ set $right l
 # Your preferred terminal emulator
 set $term alacritty
 # Your preferred application launcher
-# Note: it's recommended that you pass the final command to sway
+# Note: pass the final command to swaymsg so that the resulting window is opened
```

Comment:
I think it would be better to write "can be opened". If there is configuration to bind the window to a certain workspace then it will do that instead.

Also on the following line I would put "that the command" instead of just "the command".